packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -248,11 +248,17 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
     // Get the latest `encryptedVote` for each (identifier, time).
     const encryptedVoteMap = {};
     for (const ev of encryptedVoteEvents) {
-      encryptedVoteMap[toPriceRequestKey(ev.returnValues.identifier, ev.returnValues.time)] =
-        ev.returnValues.encryptedVote;
+      encryptedVoteMap[
+        // Ancillary data is undefined or null in these "returnValues" so we add a default string of "0x".
+        // If not added, the key will improperly index and the votes will not be found.
+        toPriceRequestKey(ev.returnValues.identifier, ev.returnValues.time, ev.returnValues.ancillaryData || "0x")
```

Comment:
this is ugly but for some reason ev.returnValues.ancillaryData is undefined, even though it was sent as 0x on the vote. This causes an issue if you dont default it to '0x'

Reply:
I experienced this also when testing the `OptimisticOracleClient` and I think using the `var || 0x` is better than how I addressed it which was the ternary operator: `var ? var : 0x`